UKHomeOffice/cerberus-service
# COP-10600: Add tests to verify Risk matches on task overview and details page

## Description
Add tests to verify Risk matches on task overview and details page

## To Test
npm run cypress:runner

run the tests from spec `airpax-task-details.spec.js`
`Should check rule matches details on task details page`
`Should check  rule matches details for more than one version on task management page`

run test from spec `airpax-task-overview-page.spec.js`
`Should check rule matches details on task management page`


## Developer Checklist

\* Required
